Decisions.Microsoft365.Exchange/Steps/GroupSteps.cs

```diff
@@ -11,23 +13,33 @@ namespace Decisions.Microsoft365.Exchange.Steps
     [AutoRegisterMethodsOnClass(true, "Integration/Microsoft365/Exchange/Groups")]
     public class GroupSteps
     {
-        private static JsonSerializerSettings IgnoreNullValues = new()
+        private static readonly JsonSerializerSettings IgnoreNullValues = new()
         {
             NullValueHandling = NullValueHandling.Ignore
         };
         
-        public Microsoft365GroupList? ListGroups(ExchangeSettings? settingsOverride, bool filterUnified)
+        public Microsoft365GroupList? ListGroups(bool filterUnified,
+            [PropertyClassification(0, "Settings Override", "Settings")] ExchangeSettings? settingsOverride = null)
         {
             string urlExtension = (filterUnified) ? $"{Microsoft365UrlHelper.GetGroupUrl(null)}$filter=groupTypes/any(c:c+eq+'Unified')" : Microsoft365UrlHelper.GetGroupUrl(null);
             string result = GraphRest.Get(settingsOverride, urlExtension);
             
             return JsonHelper<Microsoft365GroupList?>.JsonDeserialize(result);
         }
         
-        public Microsoft365Group? CreateGroup(ExchangeSettings? settingsOverride,
-            string? description, string displayName, string[]? groupTypes, bool mailEnabled, string mailNickname,
-            bool securityEnabled, string[]? ownerIds, string[]? memberIds)
+        public Microsoft365Group? CreateGroup(string? description, string displayName, string[]? groupTypes,
+            bool? mailEnabled, string mailNickname, bool? securityEnabled, string[]? ownerIds, string[]? memberIds,
+            [PropertyClassification(0, "Settings Override", "Settings")] ExchangeSettings? settingsOverride)
         {
+            if (string.IsNullOrEmpty(mailNickname))
+            {
+                throw new BusinessRuleException("mailNickname cannot be null or empty. Please set a unique nickname.");
```

Comment:
Let's make this readable. `Mail Nickname cannot be null or empty. Please set a unit Mail Nickname.`